.github/workflows/build.yml

```diff
@@ -4,6 +4,7 @@ on:
     branches:
       - main
       - branch-*
+      - 636-*
```

Comment:
I assume this line is for testing and will be reverted before merging?

Reply:
Fixed